examples/config/cosmodc2_config.yml

```diff
@@ -16,7 +16,7 @@ TXCosmoDC2Mock:
     flip_g2: True # to match metacal
 
 TXIngestRedmagic:
-    lens_zbin_edges: [0.0,0.2,0.4]
+    lens_zbin_edges: [0.1, 0.3, 0.5, 0.7, 0.9]
```

Comment:
Is this related to this PR?

Reply:
Not specifically, but the [0.0,0.2,0.4] edges didn't make much sense given the N(z) of the sample so I changed it. I can change it back though, as you prefer :)

Reply:
no, this is useful